make <package> without specifying tarball is no longer working

Steps to reproduce:
- `make securedrop-client`
- `PKG_VERSION=0.4.0 make securedrop-client`

That command fails with the following output:
```
user@dev:~/src/securedrop-debian-packaging$ PKG_VERSION=0.4.0 make securedrop-client
WHEELS_DIR="/home/m/src/securedrop-debian-packaging/localwheels/" PKG_NAME="securedrop-client" ./scripts/build-debianpackage
gpg: key 310F561200F4AD77: public key "SecureDrop Release Signing Key" imported
gpg: Total number processed: 1
gpg:               imported: 1
gpg: Signature made Tue 02 Mar 2021 04:52:26 PM EST
gpg:                using RSA key 22245C81E3BAEB4138B36061310F561200F4AD77
gpg: Good signature from "SecureDrop Release Signing Key" [unknown]
gpg:                 aka "SecureDrop Release Signing Key <securedrop-release-key@freedom.press>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 2224 5C81 E3BA EB41 38B3  6061 310F 5612 00F4 AD77
Checking that SHA256SUMs from mirror match signed file...  OK
securedrop-client is a Python package
PKG_PATH not set, building from source (version 0.4.0)...
Cloning into '/tmp/securedrop-client'...
remote: Enumerating objects: 155, done.
remote: Counting objects: 100% (155/155), done.
remote: Compressing objects: 100% (116/116), done.
remote: Total 9920 (delta 67), reused 85 (delta 35), pack-reused 9765
Receiving objects: 100% (9920/9920), 7.52 MiB | 6.55 MiB/s, done.
Resolving deltas: 100% (7307/7307), done.
./scripts/build-debianpackage: line 86: PKG_GITREF: unbound variable
make: *** [Makefile:9: securedrop-client] Error 1

```
setting the `PKG_PATH` does allow the packages to be built, this appears to be a regression from when this was introduced in https://github.com/freedomofpress/securedrop-debian-packaging/pull/213

--- a/scripts/build-debianpackage
+++ b/scripts/build-debianpackage
@@ -83,7 +83,7 @@ function build_source_tarball() {
     rm -rf "$build_dir"
     git clone "$repo_url" "$build_dir"
 
-    if [[ -n "$PKG_GITREF" ]]; then
+    if [[ -n "${PKG_GITREF:-}" ]]; then
         # Can't expect a prod sig on a gitref, likely a feature branch
         git -C "$build_dir" checkout "$PKG_GITREF"
     else

